### PR TITLE
[simd] Add spec tests for bitwise operations

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@AndrewScheidecker, @tlively, @arunetm, PTAL. thanks!

File: Test/spec/simd/simd_bitwise.wast [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+  (func (export "or") (param $0 v128) (param $1 v128) (result v128) (v128.or (local.get $0) (local.get $1)))
+  (func (export "xor") (param $0 v128) (param $1 v128) (result v128) (v128.xor (local.get $0) (local.get $1)))
+  (func (export "bitselect") (param $0 v128) (param $1 v128) (param $2 v128) (result v128)
+    (v128.bitselect (local.get $0) (local.get $1) (local.get $2))
+  )
+)
+
+;; bitwise operations
+
+;; i8x16
```
Comment (AndrewScheidecker) [inline Test/spec/simd/simd_bitwise.wast]: I think my comment [here](https://github.com/WAVM/WAVM/pull/186#discussion_r321981103) also applies: it doesn't really add much to test these instructions on different forms of the `v128.const` constructor.

Comment (Honry) [reply Test/spec/simd/simd_bitwise.wast]: OK, I will remove these tests.

Comment (Honry) [reply Test/spec/simd/simd_bitwise.wast]: Done.

**Pull request merged**